src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -8,7 +8,7 @@
 
 from astroid import ImportFrom, NodeNG  # type: ignore
 
-from databricks.labs.ucx.source_code.base import Advisory
+from databricks.labs.ucx.source_code.base import Advisory, CurrentSessionState
```

Comment:
revert changes to this file.

Reply:
we need a session state to compute dependencies that rely on named parameters, so not sure it's possible to not change this file

Reply:
Sure, but what do you propose?

Reply:
And is it correct to NOT share context between files that belong to the same graph ? IIUC DB executes notebooks synchronously so parallelizing linting could introduce all sorts of issues ?